crates/cxx-qt-gen/src/syntax/foreignmod.rs

```diff
@@ -63,7 +63,7 @@ fn verbatim_to_foreign_type(tokens: &TokenStream) -> Result<Option<ForeignItemTy
             })?;
 
             Ok(Some(parse_quote! {
-                #(#attrs)*
+                #(#attrs)* // TODO: Should these be checked at this point? Passthrough of random attrs could cause problems
```

Comment:
in theory this is in the `syntax` module which is used by the `parser` module. So these are just helpers for `syn` and are not tightly bound to the CXX-Qt syntax.

So here i think it would be fine to have all attributes go through unchecked, as whatever is using this method should be checking the resultant `ForiegnItemType` and parsing that elsewhere

Reply:
Cool, removed comment